src/test/java/com/jcabi/github/mock/MkReleaseAssetTest.java

```diff
@@ -181,27 +178,24 @@ public void canPatchJson() throws Exception {
     }
 
     /**
-     * MkReleaseAsset should be able to fetch its raw representation.
+     * Should return the Base64-encoded value of the input contents. When
+     * decoded, should be equal to the input.
      *
-     * @throws Exception if some problem inside
+     * @throws Exception Unexpected.
```

Comment:
@llorllale Leave it as it was or say "If something goes wrong" -- I don't think you can ever say in a unit test that an exception is "unexpected", since there will always be an exception if it fails. Or at least it sounds strange to me :)

Reply:
@amihaiemil ok. I guess it's a question of preference